src/components/header/ProfileThreadHeaderBar.js

```diff
@@ -201,20 +212,27 @@ class ProfileThreadHeaderBar extends PureComponent<Props> {
   }
 }
 
-export default connect((state: State, props) => {
-  const threadIndex: ThreadIndex = props.index;
-  const selectors = selectorsForThread(threadIndex);
-  const selectedThread = getSelectedThreadIndex(state);
-  return {
-    thread: selectors.getFilteredThread(state),
-    threadName: selectors.getFriendlyThreadName(state),
-    processDetails: selectors.getThreadProcessDetails(state),
-    callNodeInfo: selectors.getCallNodeInfo(state),
-    selectedCallNodeIndex:
-      threadIndex === selectedThread
-        ? selectors.getSelectedCallNodeIndex(state)
-        : -1,
-    isSelected: threadIndex === selectedThread,
-    threadIndex,
-  };
-}, actions)(ProfileThreadHeaderBar);
+export default connect(
+  (state: State, props) => {
+    const threadIndex: ThreadIndex = props.index;
+    const selectors = selectorsForThread(threadIndex);
+    const selectedThread = getSelectedThreadIndex(state);
+    return {
+      thread: selectors.getFilteredThread(state),
+      threadName: selectors.getFriendlyThreadName(state),
+      processDetails: selectors.getThreadProcessDetails(state),
+      callNodeInfo: selectors.getCallNodeInfo(state),
+      selectedCallNodeIndex:
+        threadIndex === selectedThread
+          ? selectors.getSelectedCallNodeIndex(state)
+          : -1,
+      isSelected: threadIndex === selectedThread,
+      threadIndex,
+    };
+  },
+  {
+    changeSelectedThread,
+    updateProfileSelection,
+    changeRightClickedThread,
```

Comment:
don't we miss `changeSelectedCallNode` ?
(Flow doesn't warn for this, this is the biggest issue we have with react-redux&flow at the moment)

Reply:
Ugh. Thanks for catching this.